README.md

```diff
@@ -103,4 +103,28 @@ $ docker build --target test --tag my-test-image .
 Run the test container using the following command:
 ```bash
 $ docker run --env-file .env.test  my-test-image
-```
\ No newline at end of file
+```
+## Manual Deployment
```

Comment:
The only extra thing I'd love to see here is a link to your actual website, which is genuinely often something that's forgotten to be documented! <https://ka-todoapp.azurewebsites.net/>